contrib.json

```diff
@@ -202,8 +202,11 @@
         [ "git pull upstream stable",                "Update the release branch" ],
         [ "npm install",                             "Ensure dependency updates have been installed" ],
         [ "grunt test",                              "Run tests" ],
-        [ "grunt chg-release:{{ release_type }}",    "Update the changelog with the new release" ],
         [ "grunt version:{{ release_type }}",        "Bump package versions" ],
+        { "exec": "grunt vjs-version",
+          "id": "version",                   "desc": "Return the current VJS Version from the package.json file"},
+        [ "grunt chg-release:{{ version }}",         "Update the changelog with the new release" ],
+        [ "grunt clean:dist",                        "Clean out the dist folder before the build" ],
```

Comment:
It would be nice to clean out build/ as well. We have a number of vjs releases in npm with old video.js zips in the build folder.